### PR TITLE
feature(front/edit page): add cooldown countdown to submit button

### LINKED ISSUE #57: Add timer on the modification button

### PR DESCRIPTION
**Pull request by ashpieboop**
Closes #57

### CODE REVIEW

File: front/pages/edit.vue [hunk 1]
```diff
@@ -835,6 +856,7 @@ export default {
     this.fouloscopieSdk = await fouloscopie();
     this.token = this.fouloscopieSdk.userInfo.token;
     this.maxCooldownTime = await this.FetchCooldown();
+    console.log('cooldown set to', this.maxCooldownTime);
```
Comment (Sorikairox) [inline front/pages/edit.vue:859]: not useful :D

File: front/pages/edit.vue [hunk 2]
```diff
@@ -783,7 +804,7 @@ export default {
         },
       });
       const body = await res.json();
-      return body.cooldown;
+      return body.cooldown * 60 * 1000;
```
Comment (Sorikairox) [inline front/pages/edit.vue:807]: what is the purpose of this change ?

Comment (ashpieboop) [reply front/pages/edit.vue:807]: It's to convert the returned time from minutes to milliseconds.
Would it be better to have it done backend-side?

File: front/pages/edit.vue (hunk 1 above)
Comment (ashpieboop) [reply front/pages/edit.vue:859]: alrighty

**Review by Sorikairox — APPROVED** [review]

**Review by Sorikairox — APPROVED** [review]
